dev/dependencyList

```diff
@@ -27,6 +27,7 @@ arrow-memory-core/16.0.0//arrow-memory-core-16.0.0.jar
 arrow-memory-netty-buffer-patch/16.0.0//arrow-memory-netty-buffer-patch-16.0.0.jar
 arrow-memory-netty/16.0.0//arrow-memory-netty-16.0.0.jar
 arrow-vector/16.0.0//arrow-vector-16.0.0.jar
+audience-annotations/0.13.0//audience-annotations-0.13.0.jar
```

Comment:
this does not seem to be used at runtime, can we exclude it?

Reply:
It is ok to exclude this.